docs/src/components/productToolpad/ToolpadHero.js

```diff
@@ -36,7 +35,7 @@ export default function ToolpadHero() {
           </Typography>
           <GetStartedButtons
             installation="install intructions here"
-            to={ROUTES.dataGridDocs}
```

Comment:
shall we create our own `./src/route` file to centralize toolpad specific docs routes in?